app/Http/Controllers/ItemInventoryController.php

```diff
@@ -27,6 +30,14 @@ public function index(
                     )
                 );
             },
+            'export' => function () use ($request) {
+                return Excel::download(
+                    new ItemInventoriesExport($request->all() + [
+                        'user_id' => $request->user()->id,
+                    ]),
+                    'inventories-' . Carbon::now()->unix() . '.xlsx'
```

Comment:
Namanya filenya emang mau `inventories`? Bukannya ini untuk fitur `item-inventories` ya?

Reply:
Oh iya benar pak, fixed, thanks for review pak